pkg/apiserver/registry/security/workloadidentity/storage/tableconvertor.go

```diff
@@ -0,0 +1,72 @@
+// SPDX-FileCopyrightText: 2024 SAP SE or an SAP affiliate company and Gardener contributors
+//
+// SPDX-License-Identifier: Apache-2.0
+
+package storage
+
+import (
+	"context"
+	"strings"
+
+	"k8s.io/apimachinery/pkg/api/meta"
+	metatable "k8s.io/apimachinery/pkg/api/meta/table"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	metav1beta1 "k8s.io/apimachinery/pkg/apis/meta/v1beta1"
+	"k8s.io/apimachinery/pkg/runtime"
+	"k8s.io/apiserver/pkg/registry/rest"
+
+	"github.com/gardener/gardener/pkg/apis/security"
+)
+
+var swaggerMetadataDescriptions = metav1.ObjectMeta{}.SwaggerDoc()
+
+type convertor struct {
+	headers []metav1beta1.TableColumnDefinition
+}
+
+func newTableConvertor() rest.TableConvertor {
+	return &convertor{
+		headers: []metav1beta1.TableColumnDefinition{
```

Comment:
Should we also add the Subject to this table?

Reply:
Good point, but to keep the output simple, I think we should add it only when the `-o wide` flag is set, what do you think?

Reply:
To be honest, I think that the subject (the actual identity) is more important than the audiences and provider since without it the others do not make much sense. I am fine either way, but what about having Sub and Provider as default and audiences shown with `-o wide`?

Reply:
Also, now that I look at the API https://github.com/gardener/gardener/blob/master/docs/proposals/26-workload-identity.md#api-changes does it make sense "Provider" to be replaced with "TargetSystem", because logically speaking the "provider" of the identity is always Gardener.

Reply:
I changed the field `PROVIDER` to `TYPE`, added new field `SUB`, and set `AUDIENCES` to be shown only when `-o wide` is set.